Prism/PrismForm.Interface.cs

```diff
@@ -394,10 +395,13 @@ private static bool DoesEntryMatchFilter(object entry, string filter)
 
 			var meta = GetAssetMetaData(entry);
 
-			if (ulong.TryParse(filter, NumberStyles.HexNumber, Thread.CurrentThread.CurrentCulture, out var filterUid) && filterUid == meta.Uid)
+			if (filter.Contains("#") && ulong.TryParse(filter.Substring(filter.LastIndexOf('#') + 1), NumberStyles.HexNumber, Thread.CurrentThread.CurrentCulture, out var filterUid) && filterUid == meta.Uid)
```

Comment:
if including the prefix check restricts search to only searching things of that prefix type, it's probably better UX to require the prefix at the very beginning of the string instead of allowing text before it -- a search for `Iana #1234` would return all UIDs matching 1234 but nothing related to Iana, could be confusing to use

Reply:
Should be way better with the search groups now, it also checks for start characters instead of last occurences. Search might be slow with more than 2 groups if entry count is high.